lib/src/base-styles/_nx-tile.scss

```diff
@@ -45,10 +42,9 @@ $-tile-header-height-actions: 43px; // Height with sub-title and/or right action
   // NOTE: without the min-content sizing on the rows it adds extra vertical space when only title and actions are
   // present. I don't have an explanation for this, it doesn't happen with _any_ other combination of header children
   grid-template:
-    "title      tags   actions" min-content
+    "title      tags   actions" 40px
     "subtitle subtitle actions" min-content /
```

Comment:
I don't think we need or want the `actions` area to extend to this row anymore.  Which begs the question, should that last cell in the grid become part of the subtitle, or should it become unused?

Reply:
> I don't think we need or want the `actions` area to extend to this row anymore. Which begs the question, should that last cell in the grid become part of the subtitle, or should it become unused?

After talking with Jason, it was decided that it should become unused: I made an amendment here: 7dff084045a5732059fceacaf5037c260e287d8a

I used "_" to indicate unused grid-area, let me know your thoughts!

Reply:
> I used "_" to indicate unused grid-area, let me know your thoughts!

Actually CSS has a defined way to label unused areas: `.`

Reply:
> > I used "_" to indicate unused grid-area, let me know your thoughts!
> 
> Actually CSS has a defined way to label unused areas: `.`

Oh thanks! Changed it here: 62da7e966a7d58dcf57ec57c78e444825fbd33db